reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -95,8 +95,7 @@ def prepublish(self, spec, parameters, context):
 
     def submit(self, spec, parameters, state, metadata):
         """Submit a yadage packtivity to RJC."""
-        parameters = contextualize_parameters(parameters,
-                                              state)
+        # parameters = contextualize_parameters(parameters, state)
```

Comment:
Hmm, is it OK to comment out parameter handling? Will workflow parameters work in this case?  

You can test for example:

```
$ cd reana-demo-root6-roofit
$ reana-client create -n yadpar -f reana-yadage.yaml
$ reana-client upload -w yadar
$ reana-client start -w yadpar -p events=1000
...
```

and see if the workflow was run with 1000 events only instead of the default 20000.

Reply:
hm no this should not be commented out

Reply:
Is there another function that deals with parameter handling? 
I assumed since `contextualize_parameters` does not exist anymore, the default `parameters` are 'good' to go.

Reply:
Seems like it was renamed:

```console
$ cd packtivity
$ git log -p | grep -A 1 'def contextualize_para' | head -2
-def contextualize_parameters(parameters, state):
+def model_parameters(parameters, state):
```

Did  `-p events=1` work when you removed the function?

Anyway you can also experiment with replacing the function by `model_parameters()`... Haven't checked if the usage changed though.